src/ray/object_manager/plasma/stats_collector.cc

```diff
@@ -24,10 +24,17 @@ namespace plasma {
 void ObjectStatsCollector::OnObjectCreated(const LocalObject &obj) {
   const auto kObjectSize = obj.GetObjectInfo().GetObjectSize();
   const auto kSource = obj.GetSource();
+  const auto &kAllocation = obj.GetAllocation();
+
+  if (kAllocation.fallback_allocated) {
+    bytes_by_loc_seal_.Increment({/*fallback_allocated*/ true, /*sealed*/ false},
+                                 kObjectSize);
+  } else {
+    bytes_by_loc_seal_.Increment({/*fallback_allocated*/ false, /*sealed*/ false},
```

Comment:
```suggestion
    bytes_by_loc_seal_.Increment({kAllocation.fallback_allocated, /*sealed*/ false},
```

Reply:
oh lol - somehow didn't realize it. Good catch.